### ISSUE TITLE
A11y_.NET Core_ASP.NETCore_AngularFetchData_Zoom: Two scroll bars appear at 400% zoom on the page.

### ISSUE THREAD

**Issue by Truptik12** [dotnet/aspnetcore#42345]
![42345_A11y_ NET Core_ASP NETCore_AngularFetchData_Zoom](https://user-images.githubusercontent.com/86710491/175492479-f0a6375d-8e05-4d00-9e8b-478f6d64dfba.png)

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**:#A11yMAS;#A11yWCAG2.1;#A11yTCS;#.NET Core[aspnetcore]-Win32-June2022;#.NET Core;#A11ySev2;#WCAG1.4.10;#DesktopWeb;#Win11;#Benchmark;#Zoom#ceaccreviewed;#A11ycaioh;

### Environment Details:
.NET Core
ASP.NET Core
Operating System: Windows 11 Enterprise 21H2
Chrome: Version 102.0.5005.115 (Official Build) (64-bit)

### Prerequisite:
Set resolution to 1280*1024

### Steps to Reproduce:

1. Open command prompt
2. Create a folder using command "mkdir foldername" e.g. "mkdir Angular"
3. Move to the created directory using command "cd foldername"
4. Run the following command "dotnet new angular -au Individual" and hit enter.
5. Run the following command "dotnet run"
6. Copy the First URL link from Command prompt and copy it
7. Paste it in URL section of Browser and hit Search
8. Browser page will open with above template loaded
9. Hit Enter on fetch data link .
10. Zoom the screen to 400% and observe that there are two scroll bars.

### Actual:
Two scroll bars appear at 400% zoom on the page.

**This issue is also observe in the below scenario**
**1)ASP.NET Core - React Fetch Data**

1. Open command prompt
2. Create a folder using command "mkdir foldername" e.g. "mkdir React"
3. Move to the created directory using command "cd foldername"
4. Run the following command "dotnet new react -au Individual" and hit enter.
5. Run the following command "dotnet run"
6. Copy the First URL link from Command prompt and copy it
7. Paste it in URL section of Browser and hit Search
8. Browser page will open with above template loaded
9. Hit Enter on fetch data link .
10. Zoom the screen to 400% and observe that there are two scroll bars.

### Expected:
Two scroll bars should not appear at 400% zoom on the page.

### User Impact:
Low vision users will face difficulty to get the information.

**Comment by SteveSandersonMS** [comment]
@Truptik12 Could you clarify what makes this objectively a bug? I didn't see any failure notification from Accessibility Insights for this, so I'm unsure which written rule you are saying has been violated.

There's always going to be *some* level of zoom (and some window size) at which a horizontal scrollbar appears. For example here's `microsoft.com` zoomed sufficiently to get a horizontal scrollbar:

![image](https://user-images.githubusercontent.com/1101362/180977509-4b1cbef8-0871-461d-81a3-793b16f3ad14.png)

It's not clear to me at what particular zoom level we define this to be an accessibility problem. Is there an agreed definition documented somewhere saying what is the window size, screen DPI, and zoom level that needs to be handled without horizontal scrolling?

**Comment by Truptik12** [comment]
@SteveSandersonMS 

According to the reflow criteria , When user zooms up to 400% content should demand user to scroll only in one direction. The content should not be hidden or overlapped, functionality and elements on the page should not be missed.
Having to scroll in two dimensions to view content on a page makes seeing and reading the content difficult. If a user has to scroll in order to see a full line of text, it becomes very difficult to then follow to the next line.

This helps the below users:
Users with low vision who magnify their screens.
Users who use mobile devices.

**Comment by SteveSandersonMS** [comment]
> According to the reflow criteria

Thanks for the info. Are you referring to https://www.w3.org/WAI/WCAG21/Understanding/reflow.html? 

Does "Accessibility Insights for Web" somehow report this failure? I wasn't able to find any menu options inside Accessibility Insights that said anything about this criteria.

**Comment by Truptik12** [comment]
Yes I was referring to the https://www.w3.org/WAI/WCAG21/Understanding/reflow.html.

I don't think "Accessibility Insights for Web" would be able to get this bug. This bug was reported through manual testing.

**Comment by SteveSandersonMS** [comment]
Fixed in https://github.com/dotnet/spa-templates/pull/70

**Comment by mkArtakMSFT** [comment]
#Closed;
